webapp/public/style_domjudge.css

```diff
@@ -391,10 +391,13 @@ tr.ignore td, td.ignore, span.ignore {
 }
 
 .banner {
-    width: 98%;
+    max-width: 98%;
+    max-height: 100px;
```

Comment:
How does this look on a retina display? Shouldn't the max height also be a percentage or given in `x`'s maybe?

Reply:
The screenshot was from a retina screen. I don’t get what you mean with `x` but using a percentage won’t work: that’s a percentage of the page, not the image

Reply:
For percentage: fair enough. I guess in CSS it's `ex`, the height of the letter x in the current font. Just so things scale relative to the size of the rest on the page.
But TBH this is somewhat nitpicky.

Reply:
I will change it to some ex them. Or em, that’s also a thing

Reply:
But `em` is the unit size of the horizontal width of an m. So for vertical sizes, `ex` makes more sense. Although this is nitpicking in practice. :-)

Reply:
I used 12ex, which is about 100px on my machine.